upbound_uxp/ddev.yaml

```diff
@@ -0,0 +1,8 @@
+integration_tests:
+  - name: upbound_foo
+    runner: agent
+    env:
+      KUBECONFIG: /tmp/uxp.kubeconfig
+      DD_API_KEY: ${DATADOG_API_KEY}
```

Comment:
Confidence check: @ofek is this a standard pattern?

Reply:
Seems that the automated Github tests do not inject a DD_API_KEY. Please advise.

Reply:
> is this a standard pattern?

No it's not, also it looks like this file is unused so it can be removed